app/views/tickets/index.html.erb

```diff
@@ -1,7 +1,26 @@
 <div class='container'>
-  <%= link_to 'All', tickets_path %> | 
-  <%= link_to 'New', tickets_path(status: :unassigned) %>
-
+  <ul class='nav nav-tabs pt-29'>
+    <li class=<%= add_active(@search) %>><%= link_to 'All', tickets_path %></li>
+    <li class=<%= add_active(params[:status]) %>><%= link_to 'New', tickets_path(status: :unassigned) %></li>
+  </ul><br />
+  <% if @search %>
+    <div class='search pull-left' %>
+      <%= search_form_for @search do |f| %>
+        <%= f.label :email_eq, 'Search by Email : ' %>
+        <%= f.email_field :email_eq %>
+        <%=  f.submit 'Search', class: 'btn btn-primary' %>
+      <% end %>
+    </div><br />
+    <div class='pull-right mt-neg15'>
+      <%= search_form_for @search do |f| %>
+        <%= f.label 'Assignee: ' %>
+        <%= f.select :admin_id_eq, options_for_select(company_admins, params[:q][:admin_id_eq]) %>
+        <%= f.label 'Status: ' %>
```

Comment:
we don't need `params[:q][:admin_id_eq]`. It should get populated automatically.